client/app.js

```diff
@@ -29,7 +28,7 @@ const App = () => {
 
       <Switch>
         <Route exact path="/" component={WelcomePage} />
-        {/* <Route exact path= '/allProducts' component={AllProducts} />  ***NEED TO IMPORT THIS ON TOP WHEN RECEIVED***   */}
+        <Route exact path="/allProducts" component={AllProducts} />
```

Comment:
Can you change this to "products" instead of "all products"?  Just make sure products/:id is above it.